zproject/test_extra_settings.py

```diff
@@ -24,7 +24,10 @@
 FAKE_EMAIL_DOMAIN = "zulip.testserver"
 
 # Clear out the REALM_HOSTS set in dev_settings.py
```

Comment:
Added in commit https://github.com/zulip/zulip/commit/8a46ddf9d77767b8460e35cb09368a424d832f6b